views/home/updates.php

```diff
@@ -1,31 +1,20 @@
-<?php $title = "Xdebug: Updates"; include "include/header.php"; ?>
-		<tr>
-			<td>&nbsp;</td>
-			<td><span class="serif">
-			
-<!-- MAIN FEATURE START -->
+<?php
+XdebugDotOrg\Controller\TemplateController::setTitle('Xdebug: Updates');
 
-<h1>Updates</h1>
+function bug($nr) {
+	echo "bug <a href='http://bugs.xdebug.org/bug_view_page.php?bug_id=".
+		sprintf("%08d", $nr). "' target='_blank'>#$nr</a>";
+}
 
-<?php include "include/menu.php"; ?>
-<?php
-	function bug($nr) {
-		echo "bug <a href='http://bugs.xdebug.org/bug_view_page.php?bug_id=".
-			sprintf("%08d", $nr). "' target='_blank'>#$nr</a>";
-	}
-
-	function issue($nr) {
-		echo "issue <a href='http://bugs.xdebug.org/bug_view_page.php?bug_id=".
-			sprintf("%08d", $nr). "' target='_blank'>#$nr</a>";
-	}
+function issue($nr) {
+	echo "issue <a href='http://bugs.xdebug.org/bug_view_page.php?bug_id=".
+		sprintf("%08d", $nr). "' target='_blank'>#$nr</a>";
+}
 ?>
 
-<?php
-/*
-:'<,'>s/bug \#\(\d\+\)/<?php bug(\1); ?>/
-:'<,'>s/issue \#\(\d\+\)/<?php bug(\1); ?>/
-*/
-?>
```

Comment:
Please don't remove this, I use it as a reminder of what commands to run in VIM to format something copy and pasted from the issue tracker into updates. To be honest, I probably would want to have each of these in a separate file at some point. And all places where I read files (downloads, supporter logs, news, updates) also behind some case, as these operations are getting more and more expensive now there is more to do.

Reply:
ok!